### ISSUE TITLE
Local installation of anovos throws error while building reverse-geocoder on MacOS

### ISSUE THREAD

**Issue by mathiaspet** [anovos/anovos#351]
## Expected Behavior
Installing anovos in a fresh virtual environment completes w/o errors. 
## Current Behavior
Currently the install output contains errors when building the module reverse-geocoder (see attached file). 
[dump.txt](https://github.com/anovos/anovos/files/10057440/dump.txt)

## Steps to reproduce
On MacOs:
1. Create a new virtual environment
2. run ```pip install anovos```

### Specifications

- Version: Anovos v1.0.1
- Platform: MacOS Ventura 13.0.1, Python 3.10
- Subsystem:

**Comment by dattranm** [comment]
As discussed, @mathiaspet can you provide your local virtual env setup, as well as setting up pyspark_path locally, and try it with the main development branch of Anovos?

**Comment by dattranm** [comment]
I have tested on a fresh new virtual environment, with the same setup as yours, and both reverse-geocoder and anovos on PyPI are built successfully. 

Specifications
Version: Anovos v1.0.1
Platform: MacOS Big Sur Version 11.6.5, Python 3.10 (create using `conda create -n py310 python=3.10`)
Mac Version: Intel Chip

**Comment by mathiaspet** [comment]
I rechecked today and importing works fine for a virtual environment created using python v3.10.4 and 3.9 using 3.9.6 venv:
```python -m venv ./venv && source ./venv/bin/activate``` 

However, creating the virtual environment using PyCharm causes the error. It seems, that the log settings are more sensitive in PyCharm. The relevant error message is the following: 

``` Running setup.py install for reverse-geocoder ... done
  DEPRECATION: reverse-geocoder was installed using the legacy 'setup.py install' method, because a wheel could not be built for it. pip 23.1 will enforce this behaviour change. A possible replacement is to fix the wheel build issue reported above. Discussion can be found at https://github.com/pypa/pip/issues/8368```

Importing the module afterwards works fine in all three situations.

**Comment by ionicsolutions** [comment]
@mathiaspet Is `wheel` installed in the Python environment created by PyCharm? And what version of Python and `pip` are you using for PyCharm?

I don't think that this is a particular problem. If no wheel file can be built for `reverse-geocoder` in certain situations, that's something which needs to be addressed upstream.

**Comment by mathiaspet** [comment]
Yes, wheel is installed in version 0.37.1, pip is installed in version 22.3.1 and Python is installed in version 3.10.4
